src/main/java/uk/gov/hmcts/reform/unspec/helpers/CaseDetailsConverter.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.unspec.helpers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.unspec.model.CaseData;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class CaseDetailsConverter {
+
+    private final ObjectMapper objectMapper;
+
+    public CaseDetailsConverter(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    public CaseData to(CaseDetails caseDetails) {
```

Comment:
Should we name it `toCaseData`? I know it's in return type, but it'd make it easier to read imho.